27-cellular-connected-electronic-kiosk/README.md

```diff
@@ -0,0 +1,150 @@
+
+#### `kiosk_content_version`
+
+This is a version number for the file specified by `kiosk_content`. If you update the content, you can increment this number to cause the Python script to download and use the new content.
+
+#### `kiosk_download_time`
+
+This variable specifies the hour of the day (using 24-hour time) when the Python script should check for updates to the `kiosk_content`. If, for example, `kiosk_download_time` is set to 9, the script will check for updates to the content at 9:00 AM local time. If the content file name or version has changed, the script will download and use the new content. If this variable is set to the special value `now`, any change to the content file name or version will cause an immediate download of the new content, instead of waiting for a specific hour of the day.
+
+#### `kisok_data`
+
+This can be set to any valid JSON object. It will then be stored in the `resources` directory alongside `index.htm` in the file `data.js`. The contents of `data.js` looks like this:
+
+```js
+var data = <value of kiosk_data goes here>
+````
+
+For example, if you set `kiosk_data` to `{"message":"Hello world!"}`, `data.js` would look like this:
+
+```js
+var data = {"message":"Hello world!"}
+````
+
+Your web application can then access and use this data however you want. In this way, `kiosk_data` acts as a way of passing dynamic data to the web app without triggering a full download of the .zip file.
+
+## microSD Card Setup
+
+Before installing the Raspberry Pi into the Smart Pi case, you need to install [Raspberry Pi OS](https://www.raspberrypi.com/software/) onto the microSD card by following the steps in [this official video](https://www.youtube.com/watch?v=ntaXWS8Lk34) from the Raspberry Pi Foundation. Alternatively, you can follow text-based documentation [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/2).
+
+With the microSD card ready to go, plug it into the Pi:
+
+![microSD card installation](images/pi-sd.png)
+
+## Hardware Setup
+
+### Notecard and Notecarrier
+
+To connect the Notecard and Pi Hat to the Pi, follow [the section "Connect Your Notecard and Notecarrier" from the Blues Quickstart guide](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#connect-your-notecard-and-notecarrier).
+
+### Smarti Pi Case and Display
+
+To put together the Pi, display, and case, you will follow [this documentation from Smarti Pi](https://cdn.shopify.com/s/files/1/0793/8029/files/touch_pro_assembly_instructions.pdf?v=1640377735), with a few additional steps to accommodate the Notecard hardware.
+
+At Step 7, make sure to install the Notecarrier Pi Hat with the Notecard onto the Pi's headers:
+
+![Pi Hat on Pi's headers](images/pi_hat.jpg)
+
+Everything should fit, even with the standoffs attached to the Pi. You will also need to use the micro USB cable to connect the Notecarrier's micro USB port to one of the Pi's USB ports:
```

Comment:
What does this connection between the Notecarrier Pi and the RPi provide? (I ask as I didn't need to do it when using the shell scripts with this project)

Reply:
I guess you don't actually need it to run the kiosk app. It does allow you to use the notecard CLI utility to debug stuff on the Notecard. I will tweak this sentence a bit.